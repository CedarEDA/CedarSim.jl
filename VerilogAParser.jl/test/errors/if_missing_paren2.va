module if_missing_paren(p, n);
    analog begin
        if MTRLMOD == 0) begin

        end
    end
endmodule
