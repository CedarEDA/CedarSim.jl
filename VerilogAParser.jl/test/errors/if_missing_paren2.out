[31m[1mâ•­ ERROR [VAS27]:[22m[39m[31m Missing opening parenthesis.[39m
[31mâ”‚[39m[90m  â•­â”€â”€â”€â”€â”€ if_missing_paren2.va:3[39m
[31mâ”‚[90m3[0m [90mâ”‚[0m         if [91mâ—Œ [0mMTRLMOD == 0[33m)[0m begin
[31mâ”‚[0m  [90mâ”Š[0m            [32m([0m             [33mâ”“
[31mâ”‚[0m  [90mâ”Š[0m                  [33mto match `)` here[0m
[31mâ”‚[39m[90m  â”‚[39m
[31mâ”‚[39m[90m  â•°â”€â”€â”€â”€â”€ [39m
[31mâ•¯[39m